content/en/blog/falco-0-39-0/index.md

```diff
@@ -0,0 +1,153 @@
+|  |                    |  |                    |       |
+|  | Falco (eBPF probe) |  | Falco (kmod)       |       |
+|  +-------------------+  +-------------------+         |
+|                                                       |
+|                 +-------------------+                 |
+|                 | Node 3             |                |
+|                 |                    |                |
+|                 | Falco (modern eBPF)|                |
+|                 +-------------------+                 |
++-------------------------------------------------------+
+```
+In this example:
+* Node 1 is configured with the eBPF probe driver.
+* Node 2 uses the kmod (kernel module) driver.
+* Node 3 leverages the modern eBPF driver for cutting-edge performance.
+
+This setup gives you flexibility and ensures that each node in your Kubernetes cluster is running Falco in the most optimized way possible, without manual configuration. Simply set `driver.kind=auto` in the Helm chart and let Falco do the rest.
+
+## Breaking changes and deprecations ⚠️
+
+This version comes with breaking changes, mostly in the configuration interface
+
+### Removed command line options and equivalent configuration options
+
+We removed the already deprecated options  `-D`, `-t`, `-T` and is now possible to achieve the same result through the [Falco configuration](https://falco.org/docs/rules/controlling-rules/#via-falco-configuration-or-parameters)
+You con still use the `-o` command line option:
+  * -T <tag>: use `-o rules[].disable.tag=<tag>` instead. Turn off any rules with a `tag=<tag>`. This option can be passed multiple times. This option can not be mixed with `-t`;
+  * -t <tag>: use `-o rules[].disable.rule=* -o rules[].enable.tag=<tag>` instead. Only enable those rules with a `tag=<tag>`. This option can be passed multiple times;
+  * D <substring>: use -`o rules[].disable.rule=<wildcard-pattern>` instead. Turn off any rules with names having the substring <substring>. This option can be passed multiple times.
+
+You can find more information on breaking changes in the [tracking issue](https://github.com/falcosecurity/falco/issues/3045).
+
+### Notable Bug Fixes
+
+* Prometheus Compliant metrics: some metrics have been reworked to follow the prometheus best practices [#3319](https://github.com/falcosecurity/falco/pull/3319);
+* Fixed ebpf drivers to use the correct memory barrier primitive for ARM64, preventing to read incomplete data from the ring buffers [#2067](https://github.com/falcosecurity/libs/pull/2067);
+* Fixed an issue where `stats messages` were written to `stdout` and could mix with regular Falco event output 
+  [#3338](https://github.com/falcosecurity/falco/pull/3338);
+* `fs.path` fields now account for `dirfd`, fixing discrepancies with `fd.name` [#1993](https://github.com/falcosecurity/libs/pull/1993);
+
+### Deprecations
+
+In Falco 0.40.0 we will remove the `--cri` and `--disable-cri-async` options.
```

Comment:
It would be helpful to explain what to use instead.

Reply:
Done.